src/tests/zeroex/options.rs

```diff
@@ -38,157 +31,27 @@ async fn test() {
-                },
-                {
-                    "name": "CryptoCom",
-                    "proportion": "0",
-                },
-                {
-                    "name": "Lido",
-                    "proportion": "0",
-                },
-                {
-                    "name": "MakerPsm",
-                    "proportion": "0",
-                },
-                {
-                    "name": "KyberDMM",
-                    "proportion": "0",
-                },
-                {
-                    "name": "Component",
-                    "proportion": "0",
-                },
-                {
-                    "name": "Saddle",
-                    "proportion": "0",
-                },
-                {
-                    "name": "Uniswap_V3",
-                    "proportion": "1",
-                },
-                {
-                    "name": "Curve_V2",
-                    "proportion": "0",
-                },
-                {
-                    "name": "ShibaSwap",
-                    "proportion": "0",
-                },
-                {
-                    "name": "Synapse",
-                    "proportion": "0",
-                },
-                {
-                    "name": "Synthetix",
-                    "proportion": "0",
-                },
-                {
-                    "name": "Aave_V2",
-                    "proportion": "0",
-                },
-                {
-                    "name": "Compound",
-                    "proportion": "0",
-                },
-            ],
-            "orders": [
-                {
-                    "type": 0,
-                    "source": "Uniswap_V3",
-                    "makerToken": "0xe41d2489571d322189246dafa5ebde1f4699f498",
-                    "takerToken": "0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2",
-                    "makerAmount": "5876422636675954000000",
-                    "takerAmount": "1000000000000000000",
-                    "fillData": {
-                        "router": "0xe592427a0aece92de3edee1f18e0157c05861564",
-                        "tokenAddressPath": [
-                            "0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2",
-                            "0xe41d2489571d322189246dafa5ebde1f4699f498",
-                        ],
-                        "path": "0xc02aaa39b223fe8d0a0e5c4f27ead9083c756cc2000bb8e41d2489571d322189246dafa5ebde1f4699f498",
-                        "gasUsed": 67886,
-                    },
-                    "fill": {
-                        "input": "1000000000000000000",
-                        "output": "5876422636675954000000",
-                        "adjustedOutput": "5867409814019629688976",
-                        "gas": 101878,
-                    },
+                "gas": "127886",
+            },
+            "issues": {
+                "allowance": {
+                    "spender": "0xdef1c0ded9bec7f1a1670819833240f027b25eff",
```

Comment:
Is this spender actually correct? I would have expected this to be equal to the default allowance target? Can you set an allowance on either contract?